hearth/lib/hearth/middleware/build.rb

```diff
@@ -18,7 +18,8 @@ def initialize(app, builder:)
       # @param context
       # @return [Output]
       def call(input, context)
-        interceptor_error = context.interceptors.apply(
+        interceptor_error = Interceptor.apply(
+          interceptors: context.interceptors,
```

Comment:
Will the interceptors we pass ever differ from whats on context?  I think they never would.  Should we drop this parameter from the method and just use the interceptors that are on the context internally?

Reply:
Ah yeah I think we could.